### ISSUE TITLE
SDK Basic Gradle Game template doesn't handle assets nicely

### ISSUE THREAD

**Issue by tonihele** [jMonkeyEngine/sdk#301]
SDK Basic Gradle Game template doesn't handle assets nicely. To reproduce, just add any texture to Project assets sub project under Textures folder. Trying to load it via code results in error.
`com.jme3.asset.AssetNotFoundException: Textures/eisenwald.png (Flipped) (Mipmapped)
	at com.jme3.asset.DesktopAssetManager.loadAsset(DesktopAssetManager.java:371)
	at com.jme3.asset.DesktopAssetManager.loadTexture(DesktopAssetManager.java:391)
	at com.jme3.asset.DesktopAssetManager.loadTexture(DesktopAssetManager.java:401)
	at com.mygame.Main.simpleInitApp(Main.java:30)
	at com.jme3.app.SimpleApplication.initialize(SimpleApplication.java:239)
	at com.jme3.system.lwjgl.LwjglAbstractDisplay.initInThread(LwjglAbstractDisplay.java:130)
	at com.jme3.system.lwjgl.LwjglAbstractDisplay.run(LwjglAbstractDisplay.java:211)
	at java.base/java.lang.Thread.run(Thread.java:829)`

Also the template only lists jcenter as repositories. Maybe this should still be included since PSpeed's EX libraries can't be found in Maven. But at least the Maven should be included.

I don't know what the target version of the Gradle is but mainClassName is deprecated. Should be just mainClass.

**Comment by MeFisto94** [comment]
I've decided to actively exclude jCenter now, since apparently it WILL go down on February 1st and I don't plan on changing the template again until then. Paul has probably migrated in the meantime.

There's more to the template, for instance it didn't work for me since it had no gradle/wrapper folder containing the actual gradle bootstrapping jar, so I fixed that and moved to Gradle 7.0 using the toolchain feature to still stay on Java 8 though.

I'll also include something based on https://github.com/Scrappers-glitch/CrossPlatformModule in a separate commit as the advanced/cross-platform gradle template, Actually it would be cool to have something like that with the more recent kotlin script and using "cutting edge" gradle features, so it can survive the next few years without maintenance :D
